docs/Creating-Your-Own-EmulationStation-Theme.md

```diff
@@ -1,166 +1,153 @@
+- Copy those folders into `F:\emulationstation\\.emulationstation\roms`
+
 
+EmulationStation should now be set up with 45 (fake) games on three Systems, and ready for you to use.
 
-Emulationstation should now be set up with 45 (fake) games on 3 Systems, and ready for you to use.
+Double click `Launch Portable (Windowed).bat` to start EmulationStation.
 
-Double click `Launch Portable (Windowed).bat` to start Emulationstation.
+### Use Samba Share of EmulationStation's Themes on RetroPie
 
----
+You may also edit theme files directly on the Raspberry. However, you should use a wired network connection between your desktop and the Raspberry to enable smooth editing. Open `\\retropie\configs\all\themes` in the Windows File Explorer. You should be able to see (and edit) the currently installed themes of your EmulationStation installation. If you do not see any theme, then install the Spare theme as outlined below. For reloading a theme (see [shortcuts](#keyboard-shortcuts-in-emulationstation)) you should also have a keyboard attached to your RetroPie, and it should be in reach from your desktop system.
 
 ## Creating a Theme
 
-We're going to make the theme straight in the Portable Emulationstation's theme folder. This makes things a lot easier, because you can view a change straight away without having to move files each time.
+We're going to make the theme straight in the Portable EmulationStation's theme folder. Of course, you are free to act directly on the Samba Share, which does not require file copy operations.
 
-I have installed the portable Emulationstation on F:\ Drive, so the path to my themes folder is F:\emulationstation\\.emulationstation\themes.
+If you use the portable EmulationStation on `F:\` drive, so the path to the themes folder would be `F:\emulationstation\\.emulationstation\themes`.
 
-First, [download the full **Spare** theme from GitHub](https://github.com/mattrixk/es-theme-spare), so you have the images and fonts you need, and also so you have something to reference.
+First, [download the full Spare theme from GitHub](https://github.com/mattrixk/es-theme-spare), so you have the images and fonts you need, and also, so you have something to reference.
 
-We'll start by setting up a simple file structure, with folders for the 3 console systems; Gameboy (gb), Nintendo Entertainment System (nes) and Super Nintendo Entertainment System (snes).
+We'll start by setting up a simple file structure, with folders for the three console systems; Game Boy (`gb`), Nintendo Entertainment System (`nes`) and Super Nintendo Entertainment System (`snes`).
 
-- Within the themes folder make a folder called 'spare'.
-  - Within the 'spare' folder make a new file called `spare.xml`.
+- Within the themes folder make a folder called `spare`.
+  - In the `spare` folder make a new file called `spare.xml`.
```

Comment:
Wrong indent, adds an extra nesting level (see the image preview below).

----
<img width="733" alt="image" src="https://user-images.githubusercontent.com/31816814/231522499-bed809ca-859c-46ec-817f-9434c7d37754.png">

Reply:
Agreed. Is now a longer list, but there is a caption following showing the result.